platform/cosmos/model.go

```diff
@@ -78,7 +80,7 @@ func (a Attributes) GetWithdrawRewardValue() int64 {
 	result := int64(0)
 	for _, att := range a {
 		if att.Key == AttributeAmount {
-			value := strings.Replace(att.Value, UndelegateDenom, "", -1)
+			value := att.Value[:strings.IndexByte(att.Value, 'u')]
```

Comment:
all values on cosmos seems to have `u` identified in the beginning, since I can't easily inject `platform.Denom`, it might be easier to split it this way